modules/ROOT/pages/tuning-caching.adoc

```diff
@@ -0,0 +1,19 @@
+= How Caching Use Helps to Improve Processing Performance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Identifying key aspects of your data such as when to use cache and which cache strategy to follow, helps you to improve processing performance:
+
+* Is the data frequently called with often repeated results?
+* Can you afford eventual consistency between the data source and the cache information? At least for a while until cache refreshes.
```

Comment:
Our style is that bullet-list items be one sentence max, so, first I would break this:
```suggestion
* Can you afford eventual consistency between the data source and the cache information? +
At least for a while until cache refreshes.
```
But I think that the real question here is...:
* <Can your process continue if the data source and cache information is temporarily inconsistent while the cache refreshes?